lua/telescope/pickers/scroller.lua

```diff
@@ -80,6 +80,10 @@ scroller.top = function(sorting_strategy, max_results, num_results)
   return (num_results > max_results) and 0 or (max_results - num_results)
 end
 
+scroller.middle = function(max_results)
```

Comment:
Maybe we should have them always take the same params?! Tj wondered :laughing: You can do `(_, max_results, _)`

Otherwise looks good to me :) Thanks that was super fast btw

Reply:
Yeah consistency is nice. I will apply that

Reply:
as I expected, your luacheck doesn't like the underscore variable, lmk how do you want to proceed with this

Reply:
`_` is defined for function headers :laughing: otherwise its nil

Reply:
so just change it to nil and we should be good to go?

Reply:
Yeah

Reply:
Oh updating tests to `nil, value, nil` :)

Reply:
Yeah, I noticed

Reply:
Thanks :)